js_modules/dagster-ui/packages/ui-core/src/graph/asyncGraphLayout.ts

```diff
@@ -139,7 +173,17 @@ export function useAssetLayout(graphData: GraphData) {
 
     async function runAsyncLayout() {
       dispatch({type: 'loading'});
-      const layout = await asyncGetFullAssetLayout(graphData, opts);
+      let layout;
+      if (localStorageKey) {
```

Comment:
by providing a localStorageKey to the hook you opt into the localStorage caching version of this hook.